superduperdb/backends/mongodb/artifacts.py

```diff
@@ -19,6 +19,9 @@ def __init__(self, conn, name: str):
         self.db = self.conn[self.name]
         self.filesystem = gridfs.GridFS(self.db)
 
+    def url(self):
```

Comment:
Can you explain this?

Reply:
if someone explicitly called 

db = Datalayer(data_backed=...., ....)

regardless of CFG being whatever, 
with the above thing i was thinking we can validate if client db object == verctor service db object == cdc db object